src/exporter.js

```diff
@@ -0,0 +1,174 @@
+    }
+  }
+
+  addMedia(filename, data) {
+    this.media.push({filename, data});
+  }
+
+  addCard(front, back) {
+    const { topDeckId, topModelId, separator } = this;
+    const note_id = rand();
+
+    this._update('insert into notes values(:id,:guid,:mid,:mod,:usn,:tags,:flds,:sfld,:csum,:flags,:data)', {
+      ':id': note_id, // integer primary key,
+      ':guid': rand().toString(36), // rand(10**10).to_s(36) // text not null,
+      ':mid': topModelId, // integer not null,
+      ':mod': new Date().getTime() / 1000 | 0, // integer not null,
+      ':usn': -1, // integer not null,
+      ':tags': '', // text not null,
+      ':flds': front + separator + back, // text not null,
+      ':sfld': front, // integer not null,
+      ':csum': this._checksum(front + separator + back), //integer not null,
+      ':flags': 0, // integer not null,
+      ':data': '' // text not null,
+    });
+
+    return this._update(`insert into cards values(:id,:nid,:did,:ord,:mod,:usn,:type,:queue,:due,:ivl,:factor,:reps,:lapses,:left,:odue,:odid,:flags,:data)`, {
+      ':id': rand(), // integer primary key,
+      ':nid': note_id, // integer not null,
+      ':did': topDeckId, // integer not null,
+      ':ord': 0, // integer not null,
+      ':mod': new Date().getTime() / 1000 | 0, // integer not null,
+      ':usn': -1, // integer not null,
+      ':type': 0, // integer not null,
+      ':queue': 0, // integer not null,
+      ':due': 179, // integer not null,
+      ':ivl': 0, // integer not null,
+      ':factor': 0, // integer not null,
+      ':reps': 0, // integer not null,
+      ':lapses': 0, // integer not null,
+      ':left': 0, // integer not null,
+      ':odue': 0, // integer not null,
+      ':odid': 0, // integer not null,
+      ':flags': 0, // integer not null,
+      ':data': '' // text not null
+    });
+  }
+
+  _checksum(str) { return parseInt(sha1(str).substr(0, 8), 16); }
+
+  _update(query, obj) {
+    this.db.prepare(query).getAsObject(obj);
+    return this;
+  }
+
+  _getInitialRowValue(table, column = 'id') {
+    const query = `select ${column} from ${table}`;
+    return this._getFirstVal(query);
+  }
+
+  _updateInitialModelWith() {
+    const id = this.topModelId;
+    const models = this._getInitialRowValue('col', 'models');
+    const model = getLastItem(models);
+    model.name = this.deckName;
+    model.css = this.css;
+    model.did = this.topDeckId;
+    model.id = id;
+    models[id + ''] = model;
+    return this._update('update col set models=:models where id=1', { ':models': JSON.stringify(models) });
+  }
+
+  _updateInitialDeck() {
+    const { topDeckId } = this;
+    const decks = this._getInitialRowValue('col', 'decks');
+    const deck = getLastItem(decks);
+    deck.name = this.deckName;
+    deck.id = topDeckId;
+    decks[topDeckId + ''] = deck;
+    return this._update('update col set decks=:decks where id=1', { ':decks': JSON.stringify(decks) });
+  }
+
+  _getTemplate() {
```

Comment:
Move it to `index.js` and pass as a property of the second argument in constructor